### PR TITLE
[SRE-221] parse -v flag

### PR DESCRIPTION
**Pull request by pwilczynskiclearcode**

### CODE REVIEW

File: catalyst-uploader.go [hunk 1]
```diff
@@ -27,8 +27,13 @@ func run() int {
 	// cmd line args
 	describe := flag.Bool("j", false, "Describe supported storage services in JSON format and exit")
 	timeout := flag.Duration("t", 30*time.Second, "Upload timeout")
+	vFlag := flag.Lookup("v")
+	verbosity := flag.String("v", "3", "Log verbosity. {4|5|6}")
+
 	flag.Parse()
 
+	vFlag.Value.Set(*verbosity)
+
```
Comment (pwilczynskiclearcode) [inline catalyst-uploader.go:36]: I don't know where do we spawn `catalyst-uploader` process but we should pass `-v` in there.

**Comment by pwilczynskiclearcode** [comment]
It fails with panic: `flag redefined: v`

**Comment by mjh1** [comment]
@pwilczynskiclearcode the `v` flag should already be being read, the library seems to handle it. if you try running it locally on the main branch you should see the `-v` flag working. the problem is i don't know where the uploader is called in the real environment, i think somewhere in mist.

**Comment by pwilczynskiclearcode** [comment]
@mjh1 Right. Thanks.

https://github.com/livepeer/mistserver/blob/catalyst/src/controller/controller_storage.cpp?q=repo%3Alivepeer%2Fmistserver%20catalyst-uploader&type=code#L438-L444

@Thulinma is it easy to pass `-v` (or `LP_V` env) into catalyst-uploader in here

**Pull request closed without merging**